services/motion/builtin/builtin_test.go

```diff
@@ -1741,8 +1793,6 @@ func TestMoveOnMapNew(t *testing.T) {
 	})
 
 	t.Run("Is able to reach the zero pose with an empty motion configuration", func(t *testing.T) {
```

Comment:
this test case doesn't describe the actual behavior it is testing for

Reply:
not sure what you mean

Reply:
The test says `Is able to reach the zero pose with an empty motion configuration` but the request fails with this error:
`https://github.com/viamrobotics/rdk/pull/3475/files/4b3120ced9ee5ade1e4bfafbfdade8b42c5153ff#diff-d9c3cfa3ab55320d976620b0fc7572b952eea9659a7c9433bb0243b8275d91feR1809`

The test text should describe what actually happens in the test.